### ISSUE TITLE
 The sounds of alerts drowning out the stream. 

### ISSUE THREAD

**Issue by Octoober** [eerimoq/moblin#70]
Hi! My viewers are always complaining that the alerts (donations, new followers) are much louder than the stream sound. I've already tried turning down the sound in StreamElements to 10% but it doesn't help, the sound of alerts is still loud. Is there any solution to this problem?

**Comment by eerimoq** [comment]
Hello! All sounds from moblin are played through a speaker and picked up by the mic. So turning down the speaker volume should work.

**Comment by Octoober** [comment]
Yes, it helps, but then I can't hear the voiced donuts. I thought I could solve this problem by plugging in AirPods, but it only works if I use the headphone microphone. Can I choose to have the outgoing sound go to the AirPods but use the microphone built into the phone?

**Comment by Octoober** [comment]
It helped.
I really like your app! Thank you for your work.

**Comment by eerimoq** [comment]
Cool. Happy you like the app =)
